apps/editing-toolkit/editing-toolkit-plugin/help-center/class-help-center.php

```diff
@@ -141,7 +140,7 @@ function ( $product ) {
 			'plan'            => array(
 				'product_slug' => $plan->product_slug,
 			),
-			'is_wpcom_atomic' => boolval( $at_options ),
+			'is_wpcom_atomic' => boolval( defined( 'IS_ATOMIC' ) && IS_ATOMIC ),
```

Comment:
With the switch to the conditional, `boolval()` becomes redundant

Reply:
Thanks!